.github/workflows/main.yml

```diff
@@ -19,36 +19,37 @@ jobs:
       matrix:
         include:
           # MANDATORY CHECKS USING CURRENT DEVELOPMENT INTERPRETER
-          - python-version: 3.8.6
-            dependencies: >
-              dbus-python==1.2.16
-              pylint==2.4.4
-              psutil==5.6.7
+          - dependencies: >
+              pylint
+              python3-dbus
+              python3-psutil
+            image: fedora:32
             task: PYTHONPATH=./src make -f Makefile lint
-          - python-version: 3.8.6
-            dependencies: black==20.8b1 isort==4.3.21
+          - dependencies: >
+              black
+              python3-isort
+            image: fedora:32
             task: make -f Makefile fmt-travis
-          # MANDATORY CHECKS USING LOWEST SUPPORTED INTERPRETER
-          - python-version: 3.6.8
-            dependencies: >
-              dbus-python==1.2.4
-              pylint==2.4.4
-              psutil==5.4.3
+          # MANDATORY CHECKS USING LOWEST DEVELOPMENT INTERPRETER
```

Comment:
This should really be SUPPORTED, not DEVELOPMENT, because we never do our development this far back.

Reply:
Resolved.